builder/processor/pom.xml

```diff
@@ -44,6 +44,11 @@
             <groupId>io.helidon.builder</groupId>
             <artifactId>helidon-builder-processor-tools</artifactId>
         </dependency>
+        <dependency>
```

Comment:
The generated annotation is not part of the processor. It must be on classpath of the application being compiled. The processor is NOT on the classpath of the application being compiled. So why is it here?

Reply:
See the comments I made to the poms. Next push.

Reply:
pushed